tailwind.config.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  future: {
+    // removeDeprecatedGapUtilities: true,
+    // purgeLayersByDefault: true,
+  },
+  purge: {
+    enable: true, // To test locally.
```

Comment:
@rupl This didn't work for me.
With this set to true, I expected the purge to run regardless of environment, but it took https://github.com/UN-OCHA/hid_api/pull/161/commits/b28c66d1a7c5d0c0e976eadcce94a0be20c1520a#diff-1671da1302cd5c094dd762e76e142d8c1caf8f276eb884b86ef1630507e81ed2 `NODE_ENV=production ` to get the CSS purge happening locally for me.

Reply:
I agree your config is the sensible thing here. We'll see if we can solve this once we have resolved https://humanitarian.atlassian.net/browse/OPS-7176